docs/commands.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Commands
+---
+
+The driver comes with a set of many available commands, in addition to the commands included in the
+Appium base driver. Refer to the documentation of your Appium client for the exact syntax to call
+these commands.
+
+Please note that most of the driver-specific functionality is available using
+[Execute Methods](./execute-methods.md) instead.
+
+!!! info
+
+    Check the [Appium base driver documentation](https://appium.io/docs/en/latest/reference/commands/base-driver/)
+    for commands inherited by the XCUITest driver
+
+### `getClipboard`
+
+`POST` **`/session/:sessionId/appium/device/get_clipboard`**
+
+Gets the content of the primary clipboard on the device under test.
+See [Get/Set Clipboard](./clipboard.md) for more details
+
+#### Arguments
+
+| Name | Type |
+| :------ | :------ |
+| `contentType?` | `any` |
+
+#### Returned Result
+
+`string`
+
+The actual clipboard content encoded into base64 string. An empty string is returned if the
+clipboard contains no data.
+
+### `setClipboard`
+
+`POST` **`/session/:sessionId/appium/device/set_clipboard`**
+
+Sets the primary clipboard's content on the device under test.
+See [Get/Set Clipboard](./clipboard.md) for more details
+
+#### Arguments
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `content` | `any` | - |
+| `contentType?` | `any` | - |
+| `label?` | `string` | The content to be set as base64 encoded string. |
+
+#### Returned Result
+
+``null``
+
+### `toggleEnrollTouchId`
+
+`POST` **`/session/:sessionId/appium/simulator/toggle_touch_id_enrollment`**
+
+Toggle whether the device is enrolled in the touch ID program
+
+#### Arguments
+
+| Name | Type | Default value |
+| :------ | :------ | :------ |
+| `enabled?` | `any` | `true` |
+
+#### Returned Result
+
+``null``
+
+### `touchId`
+
```

Comment:
this method should be marked as deprecated in favour of the corresponding `mobile:` extension

Reply:
I will take a look, the respective `mobile:` extension is not listed in the docs, so I guess I'll need to add that too

Reply:
Check [this](https://github.com/appium/appium-xcuitest-driver/blob/master/docs/execute-methods.md#mobile-enrollbiometric) stuff. Both TouchId and FaceId could be described as biometric auth.

Reply:
Ah, it has a different name, that's why I couldn't find it. Thanks, will use that.